src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -43,7 +41,7 @@
                              KeepMetadata="REMOVE_ALL" />
     </ItemGroup>
     <ItemGroup Condition="'$(NativeAotSupported)' != 'true'">
-      <FilesToPackage Include="@(_CrossgenPublishFiles)"
+      <FilesToPackage Include="@(_CrossgenPublishFiles->Distinct())"
```

Comment:
If it ends up with duplicate files unexpectedly, maybe we should fix the root-cause rather than the first pick?

Reply:
I think I hit an issue at some point which caused me to add this, but it should no longer be needed. We should be using the next `ItemGroup` (`'$(NativeAotSupported)' == 'true'`).

Note that that already has a `Distinct()`, so there may be still a duplicate files thing going on here.